content/messaging/customer-chat-api/v3.3/index.mdx

```diff
@@ -385,6 +385,7 @@ System message event is a native system event sent in specific situations.
 | ---------------- | --------- | ----------------------------------------------------------------------------------------- |
 | `avatar`         | optional  |                                                                                           |
 | `email`          | optional  |                                                                                           |
+| `email_verified` | optional  |                                                                                           |
```

Comment:
```suggestion
| `email_verified` | optional  | Specifies if the customer confirmed their email address. See [Request Email Verification](#request-email-verification).                                                                                         |
```

Reply:
Please add the same note in other places and remember that the link might change